tests/fixtures/reference/.cargo/config.toml

```
@@ -38,8 +38,8 @@ ENV_VAR_NAME_3 = { value = "relative/path", relative = true }
 [future-incompat-report]
 frequency = 'always' # when to display a notification about a future incompat report
 
-# [cargo-new]
-# vcs = "none"              # VCS to use ('git', 'hg', 'pijul', 'fossil', 'none')
+[cargo-new]
+vcs = "none"              # VCS to use ('git', 'hg', 'pijul', 'fossil', 'none')
```

```suggestion
vcs = "none" # VCS to use ('git', 'hg', 'pijul', 'fossil', 'none')
```

(fix ci error: https://github.com/taiki-e/cargo-config2/actions/runs/10650634616/job/29522247723?pr=21#step:10:54)